### ISSUE TITLE
False Positive on lang-tag-1.4.4

### ISSUE THREAD

**Issue by jdavis4452** [dependency-check/DependencyCheck#3305]
False positive on lang-tag.jar - reported as cpe:/:tag_project:tag:1.4.4
```xml
<dependency>
	<groupId>com.nimbusds</groupId>
	<artifactId>lang-tag</artifactId>
	<version>1.4.4</version>
</dependency>
```

**Comment by nhumblot** [comment]
Hello,

What version of dependency-check are you using?

This looks to be solved by https://github.com/jeremylong/DependencyCheck/commit/f1a8fcf5afedf0764a97cc899231f0f2c0a443c1. Released with version `6.0.5`.

**Comment by jdavis4452** [comment]
We are on 5.3.2.
I will see about updating to 6.0.5
